app/models/book.rb

```diff
@@ -6,4 +6,6 @@ class Book < ApplicationRecord
   validates :author, presence: true
   validates :publisher, presence: true
   validates :cover_image_url, presence: true
-end
\ No newline at end of file
+
+  alias_attribute :book_id, :id
```

Comment:
What's this for? Why is the primary id of the Book table called `book_id`?

Reply:
this is for the Book component in the frontend, which is being used by three other components. it initially looked like this
![image](https://user-images.githubusercontent.com/91857581/190118788-7c59d267-909f-4e88-aedc-c12c052478e3.png)

where in the Discover component, the books are from the Book controller itself so the 'id' there in the addToList() would refer to the book itself.

whereas in the ReadingList/FinishedList components, the books come from the UserBook controller, so the 'id' in the markBook() and removeBook() refers to the UserBook. 

I thought this would be confusing if someone else read the code thats why I opted to change the names

Now that I think about it though, it was unecessary to use the alias on the book model itself since Book.id would make a lot more sense in the frontend and the one that actually needs to be aliased would be the UserBook id, that way it will read as Book.user_book_id

Am I on the right track here? or is there a better way of doing this